core/storage/suggestion/gae_models.py

```diff
@@ -459,6 +462,34 @@ def get_suggestions_waiting_too_long_for_review(
             cls.last_updated
         ).fetch(MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_ADMIN)
 
+    @classmethod
+    def get_new_suggestions_waiting_for_review(
+        cls,
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Returns new suggestions waiting for review that were
+        submitted within timespan of SUGGESTION_REVIEW_WAIT_TIME_NOTIFICATION
+        days.
+
+        Returns:
+            list(GeneralSuggestionModel). A list of new suggestions
+            matching the criteria.
+        """
+        current_time_millisecs = utils.get_current_time_in_millisecs()
+
+        threshold_datetime = datetime.datetime.utcfromtimestamp(
+            current_time_millisecs / 1000.0) - datetime.timedelta(
```

Comment:
Also use `fromtimestamp` and pass utc as the timezone for a timezone aware object

Reply:
Can you add the stack trace of the error here

Reply:
ERROR: test_get_new_suggestions_waiting_for_review (core.storage.suggestion.gae_models_test.SuggestionModelUnitTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/ash/Desktop/openSource/oppia/core/storage/suggestion/gae_models_test.py", line 1308, in test_get_new_suggestions_waiting_for_review
    suggestion_models.GeneralSuggestionModel.
  File "/Users/ash/Desktop/openSource/oppia/core/storage/suggestion/gae_models.py", line 487, in get_new_suggestions_waiting_for_review
    cls.created_on > threshold_datetime
  File "/Users/ash/Desktop/openSource/oppia/third_party/python_libs/google/cloud/ndb/model.py", line 1248, in __gt__
    return self._comparison(">", value)
  File "/Users/ash/Desktop/openSource/oppia/third_party/python_libs/google/cloud/ndb/model.py", line 1220, in _comparison
    value = self._do_validate(value)
  File "/Users/ash/Desktop/openSource/oppia/third_party/python_libs/google/cloud/ndb/model.py", line 1398, in _do_validate
    value = self._call_shallow_validation(value)
  File "/Users/ash/Desktop/openSource/oppia/third_party/python_libs/google/cloud/ndb/model.py", line 1756, in _call_shallow_validation
    return call(value)
  File "/Users/ash/Desktop/openSource/oppia/third_party/python_libs/google/cloud/ndb/model.py", line 1828, in call
    new_value = method(self, value)
  File "/Users/ash/Desktop/openSource/oppia/third_party/python_libs/google/cloud/ndb/model.py", line 3833, in _validate
    raise exceptions.BadValueError(
google.cloud.ndb.exceptions.BadValueError: DatetimeProperty without tzinfo created_on can only support naive datetimes (presumed UTC). Please set tzinfo to support alternate timezones.

Reply:
I think to tackle this bug we would need to make sure that created_on property of model should be set as timezone aware object, so that both the values will be timezone aware and can be compared https://github.com/oppia/oppia/pull/19045/files#diff-01f44d7b1b2d1be700ecfd1115d1dc9381f52404c1a334b0426e00e2f983d303R488. (This is if we want to make threshold value timezone aware.)

Reply:
If it's within the scope of this PR, use timezone aware object on `created_on` but if that is not the case, it's fine to leave it be in this case.